src/uenv/log.cpp

```diff
@@ -0,0 +1,21 @@
+#include <memory>
+
+#include "spdlog/sinks/stdout_color_sinks.h"
+#include "spdlog/sinks/syslog_sink.h"
+#include <spdlog/spdlog.h>
+
+#include "log.h"
+
+namespace uenv {
+void init_log() {
+    auto console_sink = std::make_shared<spdlog::sinks::stdout_color_sink_mt>();
+    console_sink->set_level(spdlog::level::warn);
+
+    auto syslog_sink = std::make_shared<spdlog::sinks::syslog_sink_mt>(
+        "uenv", LOG_PID, 0, false);
+    syslog_sink->set_level(spdlog::level::trace);
+
```

Comment:
We can customize logging pattern here. https://github.com/gabime/spdlog/wiki/3.-Custom-formatting#pattern-flags contains a list of all the built-in flags that can be used. Custom flags can also be added.

I'd think it might be useful to at least have:
- timestamp
- `[uenv]` or something else to signal that the logs are from uenv
- message level
- hostname

Generally I think the more the better (for tickets), but I guess not too much to overwhelm the users. The _final_ error messages should be as clear as possible, and I don't know yet if that should go through the logging or not.

I _think_ the syslog output leaves out everything except the message itself since it has its own mechanisms for adding metadata, but I'm not quite sure yet (haven't played around with it enough yet).

Reply:
https://github.com/bcumming/uenv2/pull/3/commits/96f098dc0bb265ffeb983faff654a048cf2ca6d1 explicitly sets the default spdlog pattern.

I don't have a reference for this, but I can confirm that my `journalctl`/`syslog` output only contains the message despite the pattern being set.